PaymentUrls/PaymentUrls.md

```diff
@@ -0,0 +1,77 @@
+- Many wallets support multiple currencies. Following the same standard will ensure higher adoption of our protocol.
+
+Example:
+```
+<a href="web+cardano:Ae2tdPwUPEZ76BjmWDTS7poTekAvNqBjgfthF92pSLSDVpRVnLP7meaFhVd">Donate</a>
+```
+
+## Considerations
+
+1. BIP-21 is limited to only features Bitcoin supports. A similar feature for Ethereum would, for example, also support gas as an extra parameter. BIP-21 is easily extensible but we have to take precaution to avoid different wallets having different implementations of these features as they become available on Cardano. To get an idea of some extra features that could be added, consider this (still under discussion) proposal for Ethereum: [EIP-681](https://eips.ethereum.org/EIPS/eip-681)
+
+2. Depending on the protocol registration method (see next section), browsers generally enforce a `web+` or `ext+` prefix for non-whitelisted protocols (note: `bitcoin:` was whitelisted). The prefix `ext+` is recommended for extensions, but not mandatory.
+
+## ABNF Grammar (Proposal)
+
+This is an initial, simplified protocol definition for fast implementation; it only requires an address and an optional amount parameter. As discussed above, these rules are likely to evolve in time in order to support additional features, including unique capabilities of the Cardano blockchain.
+
+```
+cardanourn     = "web+cardano:" cardanoaddress [ "?" amountparam ]
+cardanoaddress = *base58
+amountparam    = "amount=" *digit [ "." *digit ]
+```
+
+The amount parameter must follow the [same rules](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki#transfer-amountsize) described in BIP-21, namely, it must be specified in decimal ADA, without commas and using the period (.) decimal separator.
+
+## Security Considerations
+
+1. We cannot prompt the user to send the funds right away as they may not be fully aware of the URI they clicked or were redirected to. Instead, it may be better to simply pre-populate fields in a transaction.
+2. We should be wary of people who disguise “donate” links as actually opening up a phishing website that LOOKS like a wallet.
+
+# Rationale
+
+## Why not use Universal links, deep links and other non-protocol-based Solution
+
+An alternative solution to the original problem described above is to use standard URL links in combination with a routing backend system. The routing system is used to redirect to the app's URI. The advantage of this scheme is that it allows to provide a fallback mechanism to handle the case when no application implementing the protocol is installed (for instance, by redirecting to the App Store or Google Play). This is the approach behind iOS Universal Links and Android App Links. In general, it provides a better user experience but requires a centralized system which makes it unsuitable for as a multi-app standard.
+
+## Read More
```

Comment:
Maybe add a link to existing Yoroi implementations as a reference?

Reply:
Our implementation is probably too deeply coupled with the rest of our code to serve as a good reference unfortunately